src/defaultPhrases.js

```diff
@@ -0,0 +1,176 @@
+const closeDatePicker = 'Close';
+const focusStartDate = 'Focus on start date';
+const clearDate = 'Clear Date';
+const clearDates = 'Clear Dates';
+const jumpToPrevMonth = 'Jump to previous month';
+const jumpToNextMonth = 'Jump to next month';
+const keyboardShortcuts = 'Keyboard Shortcuts';
+const showKeyboardShortcutsPanel = 'Show the keyboard shortcuts panel';
+const hideKeyboardShortcutsPanel = 'Hide the keyboard shortcuts panel';
+const toggleKeyboardShortcutsPanel = 'Toggle the keyboard shortcuts panel';
```

Comment:
> Seems like you shouldn't be able to access this button when the shortcut panel is shown
I am not sure how to make that happen, but that's certainly not the behavior rn. Could you maybe run the code and lemme know what you think?